systemd-units/plymouth-poweroff.sh.in

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+if [ "$1" = "poweroff" ]; then
+	@PLYMOUTH_CLIENT_DIR@/plymouth quit --retain-splash
```

Comment:
@carlocaione I had a question about the use of this option; chatting with @magcius on IRC, it seemed to us that it's mostly used (or was it used?) for the X->GDM transition, not for this sort of use case.
Do things still work without it?